tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -303,17 +303,35 @@ class TestFicTracDataInterfaceTiming(TemporalAlignmentMixin, unittest.TestCase):
     save_directory = OUTPUT_PATH
 
 
-class TestVideoInterface(VideoInterfaceMixin, unittest.TestCase):
+import pytest
+
+
+class TestVideoInterface(VideoInterfaceMixin):
     data_interface_cls = VideoInterface
-    interface_kwargs = [
-        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_avi.avi")]),
-        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_flv.flv")]),
-        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mov.mov")]),
-        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mp4.mp4")]),
-        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_wmv.wmv")]),
-    ]
     save_directory = OUTPUT_PATH
 
+    @pytest.fixture(
+        params=[
+            (dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_avi.avi")])),
+            (dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_flv.flv")])),
+            (dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mov.mov")])),
+            (dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mp4.mp4")])),
+            (dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_wmv.wmv")])),
+        ],
+        ids=["avi", "flv", "mov", "mp4", "wmv"],
+    )
+    def setup_interface(self, request):
```

Comment:
I think our eventual preferred syntax was actually separate classes altogether, like

```python
class TestVideoInterfaceAVI(VideoInterfaceMixin):
    interface_kwargs = {"file_path": str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_avi.avi")}
```

That way you don't need the extra `setup_interfaces` step, and you can attach separate docstrings to each class (I personally find that very useful when adding new test cases for bug fixes since I can link to the original issue in the docstring as a reminder)

Technically a little bit more writing, but better verbosity and less magic overall

I find pytest params are great for expanding matrices of inputs and input types in simple functions (like testing both `str` and `path` inputs in cases like this) but I think it's overkill here

Reply:
I am fine with that too. It is also necessary when the test of the interface will check for metadata.

But I will just do this refactor so the diff is minimal and then we can break classes with parameterized in multiple classes after that.